aspnetcore/fundamentals/routing.md

```diff
@@ -135,6 +135,21 @@ Routing is connected to the [middleware](xref:fundamentals/middleware/index) pip
 
 URL matching is the process by which routing dispatches an incoming request to an *endpoint*. This process is based on data in the URL path but can be extended to consider any data in the request. The ability to dispatch requests to separate handlers is key to scaling the size and complexity of an app.
 
+::: moniker-end
+
+::: moniker range=">= aspnetcore-3.0"
+
+When a Routing Middleware executes, it sets an endpoint (`Endpoint`) and route values to a feature on the <xref:Microsoft.AspNetCore.Http.HttpContext>. For the current request:
+
+* `HttpContext.GetEndpoint` gets the endpoint.
```

Comment:
```suggestion
* `HttpContext.GetEndpoint()` gets the endpoint.
```

Reply:
🤔 hum ... we don't ordinarily show paren for methods. How about this ...

```suggestion
* Calling `HttpContext.GetEndpoint` gets the endpoint.
```

Reply:
Yeah ... let's do that. We aren't going to get the paren after we use the `xref` link later anyway.

btw- I made a note on ...

**[RTM] Remove version context from Routing** #9552

... to address the links. I (or someone else) will take care of it.